src/main/java/pl/umk/mat/git2befit/controller/workout/TrainingPlanController.java

```diff
@@ -21,21 +23,28 @@ public TrainingPlanController(TrainingPlanService service) {
 
     @PostMapping("/generate")
     public ResponseEntity<?> generate(
-            @RequestBody(required = false) TrainingForm trainingForm,
-            @RequestHeader(value = "Authorization", required = false) String authorizationToken
+            @RequestBody TrainingForm trainingForm,
+            @RequestHeader(value = "Authorization", required = false) String authorizationToken,
+            @RequestHeader(value = "Date", required = false) String date
```

Comment:
```suggestion
            @RequestHeader(required = false) String date
```

Reply:
A to nie jest tak, że to jest nie czułe  na znaki?

Reply:
Sprawdzone, jest czułe na wielkości znaków.